exla/mix.exs

```diff
@@ -33,8 +33,7 @@ defmodule EXLA.MixProject do
   defp deps do
     [
       {:nx, path: "../nx"},
-      # TODO: adjust
-      {:xla, "~> 0.1.1-dev", runtime: false, github: "jonatanklosko/xla", branch: "jk-build"},
+      {:xla, "~> 0.1.0", runtime: false, github: "elixir-nx/xla"},
```

Comment:
I think it actually makes sense to point to GitHub for now, so if we need any fixes to the Elixir code, we can do so quickly. We can publish to hex at any time.